rainbowstream/rainbow.py

```diff
@@ -410,9 +411,33 @@ def tweet():
     """
     Tweet
     """
-    t = Twitter(auth=authen())
-    t.statuses.update(status=g['stuff'])
+    # Regex to check if tweet contains '--i' pattern
+    pattern = r'(.*) \-\-\i\ (.+)'
+    m = re.match(pattern, g['stuff'])
+
+    if m is None:
+        # text only tweet
+        t = Twitter(auth=authen())
+        t.statuses.update(status=g['stuff'])
+    else:
+        # A tweet with media items
+        body = m.group(1)
+        imagePaths = m.group(2)
+
+        # Generating image ids
+        imageIds = []
+        for impath in imagePaths.split(','):
+            imagedata = open(impath, 'rb').read()
```

Comment:
Should wrap this command inside a try and except block, and when it failed we can 
```python
printNicely(red('Some nice error message here'))
```

Reply:
wrapped in method called upload_twitter_media and invoked that method in try/catch
<img width="496" alt="Screenshot 2021-04-17 at 19 11 59" src="https://user-images.githubusercontent.com/1627972/115119465-1da03b00-9fb1-11eb-9155-977780951c37.png">